docs/Quick-Start.md

```diff
@@ -2,7 +2,26 @@
 sort: 13
 ---
 
-# Quick Start
+# Quick start
```

Comment:
Can we add details how to quick start with cluster version as well?

Reply:
Will do, thanks